pystiche/papers/ulyanov_et_al_2016/loss.py

```diff
@@ -0,0 +1,182 @@
+from typing import Union, Optional, Sequence, Any, Dict
+from collections import OrderedDict
+import torch
+from pystiche.enc import Encoder, MultiLayerEncoder
+from pystiche.ops import (
+    MSEEncodingOperator,
+    GramOperator,
+    MultiLayerEncodingOperator,
+    TotalVariationOperator,
+)
+from pystiche.loss import MultiOperatorLoss
+from .utils import ulyanov_et_al_2016_multi_layer_encoder
+
+
+def ulyanov_et_al_2016_content_loss(
+    impl_mode: str = "paper",
+    multi_layer_encoder: Optional[MultiLayerEncoder] = None,
+    layer: str = "relu_4_2",
+    score_weight=None,
+):
+    if score_weight is None:
+        if impl_mode == "texturenetsv1":
+            score_weight = 6e-1
+        elif impl_mode == "paper":
+            score_weight = 1e0  # FIXME paper only alpha
```

Comment:
Please address this `FIXME`.

Reply:
The paper does not provide any information on the learning rate (`lr`). This parameter is only given with a variable alpha without defined value.

Reply:
What has the learning rate `lr` to do with the content `score_weight`? If no information is given in the paper we should set it to `1e0` just as you did.